lib/pageflow/configuration/permissions.rb

```diff
@@ -0,0 +1,30 @@
+module Pageflow
+  class Configuration
+    # Permissions related options to be defined in the pageflow
+    # initializer of the main app.
+    class Permissions
+      def initialize(options = {})
+        @show_entry_features_to_non_admin =
+          options.fetch(:show_entry_features_to_non_admin, true)
+        @show_admin_boolean_to_non_admin =
+          options.fetch(:show_admin_boolean_to_non_admin, true)
+        @show_theming_dropdown_to_non_admin =
+          options.fetch(:show_theming_dropdown_to_non_admin, true)
+      end
+
+      # Allow non-admin users to see entry features. Defaults to true.
+      # @since edge
+      attr_accessor :show_entry_features_to_non_admin
```

Comment:
This one I like very much. Will do another renaming pass.

I do not have a hard time imagining permission adjustments. That may be due to my imagination or the fact that I implemented many change requests regarding permissions already -- before we merged the permissions branch